org.civicrm.bootstrapcivihr/bootstrapcivihr.php

```diff
@@ -117,3 +117,9 @@ function bootstrapcivihr_civicrm_pageRun($page) {
     CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.bootstrapcivihr', 'css/civihr.css');
   }
 }
+/**
+ * Implements hook_civicrm_buildForm().
+ */
+function bootstrapcivihr_civicrm_buildForm($formName, &$form) {
+  CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.bootstrapcivihr', 'css/civihr.css');
```

Comment:
why is this needed? I thought that the `_pageRun` hook would add the css in any page?

Reply:
Yeah but, that's true for only for CiviCRM page classified as _page_. This does not affect on page type _form_
You can check more on that at, https://docs.civicrm.org/dev/en/stable/hooks/hook_civicrm_pageRun/